cypress.config.ts

```diff
@@ -7,6 +7,4 @@ export default defineConfig({
         baseUrl: 'http://localhost:3000',
         testIsolation: false
     },
-    viewportHeight: 927,
-    viewportWidth: 1600
```

Comment:
This does not appear to be having an effect in tests.

Reply:
it's duplicate, let's strip it out 👍